internal/cmdsearch/cmdsearch.go

```diff
@@ -32,6 +32,8 @@ func NewSearchRunner(name string) *SearchRunner {
 		"Match by path expression of a field.")
 	c.Flags().StringVar(&r.PutLiteral, "put-literal", "",
 		"Set or update the value of the matching fields with the given literal value.")
+	c.Flags().StringVar(&r.PutPattern, "put-pattern", "",
+		"Set or update the value of the matching fields with the given literal value.")
```

Comment:
The description is exactly the same as `put-literal`. It should probably be updated?